internal/schema-registry/command_cluster_describe.go

```diff
@@ -20,7 +20,9 @@ import (
 type clusterOut struct {
 	Name                string `human:"Name" serialized:"name"`
 	Cluster             string `human:"Cluster" serialized:"cluster"`
-	EndpointUrl         string `human:"Endpoint URL" serialized:"endpoint_url"`
+	EndpointUrl         string `human:"Endpoint URL,omitempty" serialized:"endpoint_url"`
+	PrivateEndpointUrl  string `human:"Private Endpoint URL,omitempty" serialized:"private_endpoint_url"`
+	CatalogEndpointUrl  string `human:"Catalog Endpoint URL,omitempty" serialized:"catalog_endpoint_url"`
```

Comment:
I think we can add `omitempty` for the serialized versions of the two new fields.

Reply:
should i also add it to the already existing `EndpointUrl`?

Reply:
No, I think that would technically be a breaking change. A user might have a script that explicitly reads that field (even if it is `""`) and hiding it might break their script.

Reply:
Update for @brianstrauch: We decided to add this anyway on the following reasoning: only one will be non-empty at a time, and this change would only affect users who opt in to private (so any script that uses the public endpoint would fail anyway)

Reply:
Makes sense!